src/EditorFeatures/Core/CommandState.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.CodeAnalysis.Editor
     /// <summary>
     /// Represents the various states that a command might have.
     /// </summary>
-    internal struct CommandState
+    internal struct CommandState2
```

Comment:
So, is the pattern that the `2` is in Roslyn?  Does this impact TypeScript/Xaml/F#?

Reply:
:bulb: This structure is no longer required. Use `CommandState` instead.